package.json

```diff
@@ -36,12 +36,12 @@
     "ES2016"
   ],
   "dependencies": {
-    "babel-runtime": "5.6.20",
-    "bluebird": "^2.9.33",
+    "babel-runtime": "^5.8.29",
+    "bluebird": "^3.0.1",
     "browser-sync": "^2.7.13",
     "chokidar": "^1.0.3",
     "cli-color": "^1.0.0",
-    "co": "^3.1.0",
+    "co": "^4.6.0",
```

Comment:
Didn't this change break the config method in the past? The best course of action would be completely removing `co` I believe, but for that we have to replace co-prompt with another prompt library (e.g. Inquirer).